backend/controllers/taskController.js

```diff
@@ -19,37 +19,22 @@ const createTask = asyncHandler(async (req, res) => {
 });
 
 const getTasks = asyncHandler(async (req, res) => {
-  // console.log("hereee");
-  const header = req.headers.authorization;
-  const token = header.split(" ")[1];
-
-  const user_data = jwt.decode(token);
-
-  const userId = user_data._id;
+  const userId = req.id;
   await Task.find({ user_id: userId })
     .then((docs) => {
       res.status(200).json(docs);
     })
     .catch((err) => {
       if (err) {
```

Comment:
The `if err` check isn't needed because that's the purpose of the `.catch` function, otherwise it's just redundant

Reply:
Fixed: Changes are reflected in recent commits